lib/util/exec/containerbase.ts

```diff
@@ -171,6 +171,11 @@ const allToolConfig: Record<string, ToolConfig> = {
     packageName: 'pnpm',
     versioning: npmVersioningId,
   },
+  pixi: {
+    datasource: 'github-releases',
+    packageName: 'prefix-dev/pixi',
+    versioning: pep440VersioningId,
```

Comment:
are you sure with that versioning? 🤔 we usually use semver for such GitHub packages.

Reply:
yes: https://github.com/prefix-dev/pixi/releases/tag/v0.33.0rc1